treeherder/client/thclient/client.py

```diff
@@ -681,7 +682,16 @@ def _get_json(self, endpoint, timeout, project=None, **params):
 
         resp = requests.get(uri, timeout=timeout, params=params,
                             headers=self.REQUEST_HEADERS)
-        resp.raise_for_status()
+        try:
+            resp.raise_for_status()
+        except HTTPError as e:
+            response = e.response
+            logger.error("Error submitting data to %s" % response.request.url)
```

Comment:
The URL is always included in the HTTPError:
https://github.com/kennethreitz/requests/blob/f5dacf84468ab7e0631cc61a3f1431a32e3e143c/requests/models.py#L830-L834

...which presumably also ends up in the log as well. I don't know if you think this makes the URL here redundant?

I'm happy either way :-)